default/tests/test_main.py

```diff
@@ -1,10 +1,19 @@
+import requests
 from fastapi import FastAPI
 from fastapi.testclient import TestClient
+from unittest.mock import MagicMock
 
 from main import app, models
 
 client = TestClient(app)
 
+class MockHTTPResponse:
```

Comment:
FYI, when using `aiohttp`: https://docs.aiohttp.org/en/stable/testing.html

Reply:
Looks like it's for testing aiohttp server, not patching client

Reply:
It also has a testing client: https://docs.aiohttp.org/en/stable/testing.html#pytest_aiohttp.aiohttp_client